<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCheckHanRegWidget</class>
 <widget class="QWidget" name="CCheckHanRegWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>5</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="vlay_widget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="vlay_pam">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>参数设置</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>胶带数</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spin_tapeNum">
               <property name="value">
                <number>4</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>动态阈值</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>5</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>核大小</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spin_dynkernel">
                  <property name="value">
                   <number>30</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>动态阈值</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spin_dynThr">
                  <property name="value">
                   <number>5</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>焊点筛选</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="spacing">
               <number>5</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>直径</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spin_hanHoleSize">
                  <property name="value">
                   <number>15</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>最大面积</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spin_maxArea">
                  <property name="maximum">
                   <number>999</number>
                  </property>
                  <property name="value">
                   <number>180</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>焊点数</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spin_hanSize">
                  <property name="value">
                   <number>15</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_read">
             <property name="text">
              <string>参数读取</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btn_detect">
             <property name="text">
              <string>检测</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spin_hanHoleSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>btn_detect</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>737</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>702</x>
     <y>543</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_maxArea</sender>
   <signal>valueChanged(int)</signal>
   <receiver>btn_detect</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>752</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>714</x>
     <y>541</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_hanSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>btn_detect</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>742</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>734</x>
     <y>542</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
